cves/CVE-2017-14056.yml

```diff
@@ -342,7 +360,16 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The main mistake is trusting inputs. The chance of an attacker
+    getting someone to both download and then use an evil file just for 
+    a Denial of Service attack are low, so it is easy to see why this may have
```

Comment:
I would explain what a `demuxer` is, as I don't have the knowledge here and I'm not sure if other readers would.

Overall I think you have good learnings here, but I'm not sure how much we should assume that they didn't think a risk was high, versus just testing conditions at all. It's hard to dive into their thoughts unless there's forum posts or discussions about their methodologies,